### DISCUSSION TITLE
[Unstable] Gravity Shells generates an additional explosion in the center of the hull when explode

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#14221] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/FakeFishGames/Barotrauma/assets/116519004/686bdae2-c8f5-4ad9-bfe4-4db4f27c227f
Both of Flak and Railgun
and if you change the size of the hull, the explosion will be generated at the new center point of the hull

### Reproduction steps

just shoot some thing and try resizing the hull see what difference

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

V1.6.100.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in the latest unstable
